### ISSUE TITLE
Select basic's highlighted items is confusing.

### ISSUE THREAD

**Issue by simonswine** [grafana/design-system#152]
This is an issue that I had first created in https://github.com/grafana/explore-profiles/issues/106.

I've basically found it confusing that I can see two items highlighted in the datasource drop down of the Explore apps.

This problem also exists for https://developers.grafana.com/ui/latest/index.html?path=/story/forms-select--basic.

Here a short video:

https://github.com/user-attachments/assets/c21538cf-ee3a-43b1-abeb-adfdb30eba18

Maybe it makes sense to follow e.g [Material UI](https://mui.com/material-ui/react-select/) where the focus is on the selected item when using the keyboard.

**Comment by tskarhed** [comment]
A fix for seeing the difference between the two is in canary https://developers.grafana.com/ui/canary/index.html?path=/story/forms-select--basic

In terms of it highlighting the top option, the keyboard focus is on it, which means you only need to press enter to select it. Looking through the other issue, do I understand it correctly that the preferred behavior is that the focus defaults to the selected item? I think that makes sense.

**Comment by simonswine** [comment]
> [...] do I understand it correctly that the preferred behavior is that the focus defaults to the selected item?

Yes that seems the most natural to me (but I am also more of a backend devloper).

**Comment by tskarhed** [comment]
>  (but I am also more of a backend devloper).

And in this case a good example of a user experiencing issues ;)
